webui/src/components/community_admin.jsx

```diff
@@ -0,0 +1,316 @@
+            for(var k in this.props.roles){
+                if(this.props.users[userid]["roles"].includes(this.props.roles[k])){
+                    roleName.push(k);
+                }
+            }
+            rows.push( <UserRow userID={userid}
+                                userEmail={this.props.users[userid]["email"]}
+                                userRoleID={this.props.users[userid]["roles"]}
+                                userRoleName={roleName}
+                                key={userid}
+                                roles={this.props.roles} />);
+        });
+
+        return (
+            <div>
+                <div className="row" >
+                    <div className="col-sm-4"><h4><strong> Email Address </strong></h4></div>
+                    <div className="col-sm-2"><h4><strong> Role </strong></h4></div>
+                    <div className="col-sm-6"><h4><strong> Edit </strong></h4></div>
+                </div>
+                <div >{rows}</div>
+            </div>
+        );
+    }
+});
+
+
+export const UserRow = React.createClass({
+    render(){
+        if (!this.props.roles) {
+            return <Wait/>;
+        }
+
+        var showRolesName = this.props.userRoleName.map((role) => <div key={role}> {role || ''}</div>);
+        return (
+            <div>
+                <div className="row well">
+                    <div className="col-sm-4">{this.props.userEmail}</div>
+                    <div className="col-sm-2">{showRolesName}</div>
+                    <div className="col-sm-6"><EditRoles roles={this.props.roles} 
+                                                        defRoleName={this.props.userRoleName} 
+                                                        defRoleID={this.props.userRoleID} 
+                                                        userEmail={this.props.userEmail}
+                                                        userID={this.props.userID}/ ></div>
+                </div> 
+            </div>
+        );
+    }
+});
+
+
+
+export const EditRoles = React.createClass({
+
+    getInitialState() {
+        return { 
+            edit: false,
+            currentAssignedRole: null,
+        };
+    },
+
+    componentWillMount(){
+        this.componentWillReceiveProps(this.props);
+    },
+
+    componentWillReceiveProps(nextProps){
+        this.setState({currentAssignedRole: nextProps.defRoleName});
+    },
+
+    onSubmit(e) {
+        e.preventDefault();
+        this.editUsersRoles();
+    },
+
+    assigneRole(roleid, userEmail){
+        serverCache.registerUserRole(userEmail, roleid,
+            () => {
+                notifications.success("The new role was added to the user");  
+            },
+            () => {
+                notifications.danger("An error occured while trying to add the new role to the user");
+            }
+        );
+    },
+
+    deleteRole(roleid, userid){
+        const current_user=serverCache.getUser()
```

Comment:
@emanueldima if we want to enforce some code style maybe we could use some tools enabling it? I was already planning to add pep8 code style verification at some point. In Invenio the style is checked when `./run-tests.sh` is ran.